### ISSUE TITLE
Netflix playing error

### ISSUE THREAD

**Issue by tyshiro** [CastagnaIT/plugin.video.netflix#1698]
### Netflix add-on version

V1.23.3+matrix.1

### Operative systems used

Android

### Kodi version used

Other (specify in description)

### Description of the bug

Kodi 21 omega
Keep getting this error message when trying to play episodes: disabled due to exception:”nf_is_ads_ stream”

### Steps to reproduce the behavior

1-Open Netflix
2-Choose show want To watch
3-Click show to load
4-Receive error message – disabled due to exception:”nf_is_ads_ stream”

### Debug log - mandatory

https://paste.kodi.tv/xawovawipu

### Screenshots

![IMG_8039](https://github.com/CastagnaIT/plugin.video.netflix/assets/70737955/a2beff4a-dc8d-4447-995f-7faf4e5cf7ab)

**Comment by sarievj** [comment]
I have the exactly same issue.

**Comment by JuRo1971** [comment]
same here

**Comment by Schaagi** [comment]
For the past two days, I have been receiving the same message when starting a movie on Netflix. Playback disabled due to an exception: nf_is_ads_stream. 
Reinstalling Widevine did not help.

**Comment by JuRo1971** [comment]
I was wondering, how the plugin still worked for some guys. On the nvidia shield, the plugin does not work for years now ...

**Comment by dowerner** [comment]
I hope this helps somebody:

TLDR:
- Try to disable and re-enable the Netflix addon
- Downgrade InputStream adaptive

I was having the same issue and tried to debug by adding log statements to the offending service in the addon. To reload I disabled and re-enabled the addon several times. When trying to trigger the issue again I had the streams suddenly start with audio only. This I could fix by downgrading InputStream adaptive (important not the InputStream helper).

**Comment by osflake** [comment]
Not help
![image](https://github.com/user-attachments/assets/60716bc2-38ff-4bb3-b673-8c46688ddac5)

![image](https://github.com/user-attachments/assets/f721b6f1-6695-4404-91a6-e49e6559bae5)
